project/src/pages/not-found-page/not-found-page.tsx

```diff
@@ -1,15 +1,20 @@
 import { Link } from 'react-router-dom';
 
+import Header from '../../components/header/heaer';
+
 import { AppRoute } from '../../const';
 
 import './not-found-page.css';
```

Comment:
Замени на 
```suggestion
import './not-found-page.module.css';
```
так у тебя стили точно не пересекутся с другими в проекте